src/main/java/kr/syeyoung/dungeonsguide/features/impl/solvers/FeatureSolverTeleport.java

```diff
@@ -24,10 +24,10 @@
 
 public class FeatureSolverTeleport extends SimpleFeature {
     public FeatureSolverTeleport() {
-        super("Solver.Any Floor", "Teleport", "Shows teleport pads you've visited in a teleport maze room",  "solver.teleport");
+        super("Dungeon.Solvers.Any Floor", "Teleport", "Shows teleport pads you've visited in a teleport maze room",  "solver.teleport");
```

Comment:
Category Changes shall be discussed thoroughly

Everytime we've gone through category rework, we staffs had loong discussion on how to organize stuff